src/Microsoft.AspNetCore.ResponseCaching/ResponseCachingContext.cs

```diff
@@ -82,14 +98,24 @@ private string CreateCacheKey()
                     response.Headers[pair.Key] = pair.Value;
                 }
 
-                // TODO: Update cache headers (Age)
-                response.Headers["Served_From_Cache"] = DateTime.Now.ToString();
+                // TODO: Allow setting proxied _isProxied 
+                var age = Math.Max((DateTimeOffset.UtcNow - cacheEntry.Created).TotalSeconds, 0.0);
+                var ageString = (age > int.MaxValue ? int.MaxValue : (int)age).ToString(CultureInfo.InvariantCulture);
+                response.Headers[IsProxied ? "Age" : "X-Cache-Age"] = ageString;
```

Comment:
What is `IsProxied` for? What behaviour changes should this configure? Also, We should always be setting the `Age` header.